<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>Proteomic and bioinformatics analyses of plasma from SCI neurological improvers and non-improvers</title>
    <meta charset="utf-8" />
    <meta name="author" content="Gabriel Mateus Bernardo Harrington" />
    <script src="libs/header-attrs-2.9/header-attrs.js"></script>
    <script src="libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
    <script src="libs/jquery-1.12.4/jquery.min.js"></script>
    <link href="libs/leaflet-1.3.1/leaflet.css" rel="stylesheet" />
    <script src="libs/leaflet-1.3.1/leaflet.js"></script>
    <link href="libs/leafletfix-1.0.0/leafletfix.css" rel="stylesheet" />
    <script src="libs/proj4-2.6.2/proj4.min.js"></script>
    <script src="libs/Proj4Leaflet-1.0.1/proj4leaflet.js"></script>
    <link href="libs/rstudio_leaflet-1.3.1/rstudio_leaflet.css" rel="stylesheet" />
    <script src="libs/leaflet-binding-2.0.4.1/leaflet.js"></script>
    <script src="libs/kePrint-0.0.1/kePrint.js"></script>
    <link href="libs/lightable-0.0.1/lightable.css" rel="stylesheet" />
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">
class: center, middle, inverse, title-slide

# Proteomic and bioinformatics analyses of plasma from SCI neurological improvers and non-improvers
### Gabriel Mateus Bernardo Harrington
### Keele University
### ISCoS 2021: VIRTUAL<br>Presented: 2021-10-01<br>(updated: 2021-09-07)

---









## Speaker declaration

I declare to have no interests in the below:

- The existence of any significant financial activity or other relationship
- Financial or material compensation in relation to research and publishing
- Financial or material compensation in relation to educational activities
- Ownerships and possessions in companies related to health care (includes service provides, IT)
- Compensation for expert functions in health care and consulting health care guidance processes

.center[
<div id="htmlwidget-2a74f421c7c9defbf811" style="width:756px;height:288px;" class="leaflet html-widget"></div>
<script type="application/json" data-for="htmlwidget-2a74f421c7c9defbf811">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addTiles","args":["//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"http://openstreetmap.org\">OpenStreetMap<\/a> contributors, <a href=\"http://creativecommons.org/licenses/by-sa/2.0/\">CC-BY-SA<\/a>"}]}],"setView":[[52.884422,-3.033414],16,[]]},"evals":[],"jsHooks":[]}</script>
]

???

- What an odd compulsory speil...

---

## Prior work



- Modelling of SCI neurological outcomes with routine haematological markers ([Brown, Harrington, Hulme, et al., 2019](https://doi.org/10.1089/neu.2019.6495); [Bernardo Harrington, Cool, Hulme, et al., 2020](https://doi.org/10.1089/neu.2020.7144))
- Found markers associated with liver function added modest predictive value
- Implicates liver status/metabolic health with neurological outcomes

.center[
&lt;img src="index_files/figure-html/rsquare-plot-1.png" width="50%" /&gt;
]

---

## "Bottom-up" Proteomics - A brief overview

1. Extract consistent total protein concentration for each sample
1. Optional dimensionality reduction steps (e.g. via Proteominer&amp;trade; beads)
1. Digest proteins (Trypsin is commonly used)
1. For labelled techniques, the labels are applied
1. Samples are fed through some flavour of chromatography and into the mass spec (e.g. high-performance liquid chromatography)

--

### Techniques used:

- Isobaric tag for relative and absolute quantitation ([iTRAQ](https://en.wikipedia.org/wiki/Isobaric_tag_for_relative_and_absolute_quantitation)) and [label-free](https://en.wikipedia.org/wiki/Label-free_quantification) proteomic experiments were conducted

.center[
&lt;img src="../proteomic_talk_2020-09-24/itraq_labels.png" width="40%" /&gt;
]

???

- Highlight complexity and uncertainty in measurements - put data in context, further work needed etc.

---

## Dynamic range reduction via&lt;br&gt;Proteominer&amp;trade; beads

- Abundant proteins saturate beads, whereas less abundant proteins don't
- Once the column is washed, the excess abundant proteins are removed, and the less abundant proteins are retained, thus the dynamic range is reduced

&lt;img src="label-free_and_itraq_image_ch.png" width="699" style="display: block; margin: auto;" /&gt;

---

## Experimental design

- Plasma from SCI human patients
- Two time points:
  1. "Acute": ~2-weeks post-injury
  1. "Subacute": ~3-months post-injury&lt;sup&gt;*&lt;/sup&gt;
- Improvers are AIS C patients who experienced an AIS grade conversion
- 2 4-plex iTRAQ runs and a more recent label-free experiment with all samples

&lt;table class="table table-hover" style="margin-left: auto; margin-right: auto;"&gt;
 &lt;thead&gt;
  &lt;tr&gt;
   &lt;th style="text-align:left;"&gt; Initial AIS grade &lt;/th&gt;
   &lt;th style="text-align:left;"&gt; Improver &lt;/th&gt;
   &lt;th style="text-align:right;"&gt; “Acute” &lt;/th&gt;
   &lt;th style="text-align:right;"&gt; “Subacute” &lt;/th&gt;
  &lt;/tr&gt;
 &lt;/thead&gt;
&lt;tbody&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; A &lt;/td&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; 11 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; 9 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; C &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; Yes &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; 10 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; 9 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; C &lt;/td&gt;
   &lt;td style="text-align:left;"&gt; No &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; 7 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; 6 &lt;/td&gt;
  &lt;/tr&gt;
  &lt;tr&gt;
   &lt;td style="text-align:left;"&gt; D &lt;/td&gt;
   &lt;td style="text-align:left;"&gt;  &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; 11 &lt;/td&gt;
   &lt;td style="text-align:right;"&gt; 10 &lt;/td&gt;
  &lt;/tr&gt;
&lt;/tbody&gt;
&lt;/table&gt;

.footnote[&lt;sup&gt;*&lt;/sup&gt; iTRAQ experiments did not include AIS A or D groups at the subacute time point]

---

## iTRAQ key pathway - Complement cascade

- Total number of proteins identified: 79
- Majority of proteins identified are part of the complement cascade, differences in these pathways appears to be at least partially responsible for heterogeneous outcomes
- Many of these proteins interact with the liver in some capacity

&lt;img src="../../mateus/proteomics/figures/kegg_pathways/hsa04610_pathview.png" width="586" style="display: block; margin: auto;" /&gt;

???

The fold changes in the KEGG pathway are from the Acute AIS C improvers relative to non-improvers

---

## iTRAQ key protein identified: Serum amyloid A1

.pull-left[
&lt;img src="index_files/figure-html/saa1-plot-1.png" width="504" /&gt;
]

.pull-right[
- [Serum amyloid A1](https://www.uniprot.org/uniprot/P0DJI8)
- Precursor of amyloid A, the aberrant deposition of which leads to inflammatory amyloidosis ([Sun and Ye, 2016](https://doi.org/10.1016/j.gene.2016.02.044))
- Produced by hepatocytes, during acute phase response they associate with HDL, displacing ApoA1 ([Benditt and Eriksen, 1977](https://doi.org/10.1073/pnas.74.9.4025))
- SAA binding HDL is thought to favour removal of cholesterol from sites of inflammation
- SAA can also bind retinol, potentially limiting bacterial burden in tissues ([Derebe, Zlatkov, Gattu, et al., 2014](https://doi.org/10.7554/elife.03206))
]

???

### Proteinpilot

- way more abundant in subacute C imp vs non-imp (~28 fc)
- even more so in acute C imp vs D

### Openms
- Downreg in acute C imp vs non-imp (~ -2.5 fc)
- Upreg in subacute C imp vs non-imp (~ 2.4 fc)
- Downreg in C groups and D vs A

---

## Label-free proteomics

- Data still being analysed
- Substantial overlap in proteins identified
- Complement cascade still the most significant

- Early proteins of interest:
 - Peroxiredoxin 2, detected in Acute C improvers but not Acute C non-improvers. Involved in oxidative stress response
 - Several immunoglobulins variable and constant regions, potential link to work suggesting IVIG therapy benefiting outcomes? ([Brennan, Kurniawan, Vukovic, et al., 2016](https://doi.org/10.1002/acn3.318))

### Link between metabolic status and SCI?

- Mounting evidence to support a link between the liver, and perhaps metabolic health more broadly, and SCI, including neurological outcomes
- Excellent review by Goodus and McTigue on the topic ([Goodus and McTigue, 2020](https://doi.org/10.1016/j.expneurol.2019.113160))

???

- touch on future work
- validation at Glasgow

---

## Links to this slide deck:

The rendered slides can be found at this address:

- [https://h-mateus.github.io/presentations/iscos_sci_proteomics_2021-08-31/index.html#1](https://h-mateus.github.io/presentations/iscos_sci_proteomics_2021-08-31/index.html#1)

.center[
&lt;img src="slides_qr-code.png" width="40%" /&gt;

The raw files are on GitHub [here](https://github.com/H-Mateus/presentations) under the GPL license
]

---

## Acknowledgements

.pull-left[
- Thanks to my supervisors: Karina Wright, Paul Cool, Charlotte Hulme &amp; the rest of the [OsKOR team](https://oskor.netlify.app/)
- Thanks to the clinical team at the [Midland Centre for Spinal Injuries](https://www.rjah.nhs.uk/Our-Services/Spinal-Injuries-Unit.aspx), and all the patients who've donated to our biobank
- Thanks for the funding: [EPSRC](https://epsrc.ukri.org/)
- Thanks to the open-source community for facilitating this research, Yihui Xie's R package [xaringan](https://github.com/yihui/xaringan) in particular for these slides
]

.pull-right[
&lt;img src="../logos/EPSRC+logo.png" width="90%" /&gt;&lt;img src="../logos/keele_logo.jpg" width="90%" /&gt;&lt;img src="../logos/rjah_logo.png" width="90%" /&gt;
]

---

## References

&lt;a name=bib-benditt_amyloid_1977&gt;&lt;/a&gt;[Benditt, E. P. and N. Eriksen](#cite-benditt_amyloid_1977) (1977). "Amyloid Protein SAA Is Associated with High Density Lipoprotein from Human Serum". En. In:
_Proceedings of the National Academy of Sciences_ 74.9, pp. 4025-4028. ISSN: 0027-8424, 1091-6490. DOI: [10.1073/pnas.74.9.4025](https://doi.org/10.1073%2Fpnas.74.9.4025).

&lt;a name=bib-bernardo_harrington_routinely_2020&gt;&lt;/a&gt;[Bernardo Harrington, G. M., P. Cool, C. Hulme, et al.](#cite-bernardo_harrington_routinely_2020) (2020). "Routinely Measured Haematological Markers Can
Help to Predict AIS Scores Following Spinal Cord Injury". In: _Journal of Neurotrauma_. ISSN: 0897-7151. DOI: [10.1089/neu.2020.7144](https://doi.org/10.1089%2Fneu.2020.7144).

&lt;a name=bib-brennan_ivig_2016&gt;&lt;/a&gt;[Brennan, F. H., N. D. Kurniawan, J. Vukovic, et al.](#cite-brennan_ivig_2016) (2016). "IVIg Attenuates Complement and Improves Spinal Cord Injury Outcomes in Mice". En.
In: _Annals of Clinical and Translational Neurology_ 3.7, pp. 495-511. ISSN: 2328-9503. DOI: [10.1002/acn3.318](https://doi.org/10.1002%2Facn3.318).

&lt;a name=bib-brown_preliminary_2019&gt;&lt;/a&gt;[Brown, S. J., G. M. Harrington, C. H. Hulme, et al.](#cite-brown_preliminary_2019) (2019). "A Preliminary Cohort Study Assessing Routine Blood Analyte Levels and
Neurological Outcome after Spinal Cord Injury". In: _Journal of Neurotrauma_. ISSN: 0897-7151. DOI: [10.1089/neu.2019.6495](https://doi.org/10.1089%2Fneu.2019.6495).

---

## References cont.

&lt;a name=bib-derebe_serum_2014&gt;&lt;/a&gt;[Derebe, M. G., C. M. Zlatkov, S. Gattu, et al.](#cite-derebe_serum_2014) (2014). "Serum Amyloid A Is a Retinol Binding Protein That Transports Retinol during Bacterial
Infection". In: _eLife_ 3. Ed. by F. M. Powrie, p. e03206. ISSN: 2050-084X. DOI: [10.7554/elife.03206](https://doi.org/10.7554%2Felife.03206).

&lt;a name=bib-goodus_hepatic_2020&gt;&lt;/a&gt;[Goodus, M. T. and D. M. McTigue](#cite-goodus_hepatic_2020) (2020). "Hepatic Dysfunction after Spinal Cord Injury: A Vicious Cycle of Central and Peripheral Pathology?"
En. In: _Experimental Neurology_ 325, p. 113160. ISSN: 0014-4886. DOI: [10.1016/j.expneurol.2019.113160](https://doi.org/10.1016%2Fj.expneurol.2019.113160).

&lt;a name=bib-sun_serum_2016&gt;&lt;/a&gt;[Sun, L. and R. D. Ye](#cite-sun_serum_2016) (2016). "Serum Amyloid A1: Structure, Function and Gene Polymorphism". En. In: _Gene_ 583.1, pp. 48-57. ISSN: 0378-1119. DOI:
[10.1016/j.gene.2016.02.044](https://doi.org/10.1016%2Fj.gene.2016.02.044).
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"highlightStyle": "github",
"highlightLines": true,
"countIncrementalSlides": false
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>
<style>
.logo {
  background-image: url(../logos/iscos_corner_logo_background_removed.png);
  background-size: contain;
  background-repeat: no-repeat;
  position: absolute;
  top: 0.1em;
  right: 0.1em;
  width: 110px;
  height: 128px;
  z-index: 0;
}
</style>

<script>
document
  .querySelectorAll(
    '.remark-slide-content' +
    ':not(.title-slide)' +
    // add additional classes to exclude here, e.g.
    // ':not(.inverse)' +
    ':not(.hide-logo)'
  )
  .forEach(el => {
    el.innerHTML += '<div class="logo"></div>';
  });
</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
